### PR TITLE
Added websockets tutorial [#1925]

### PR DESCRIPTION
**Pull request by ryancreatescopy**
## Description

Added new websockets tutorial by Alchemy.

cc @samajammin can you just check the language declarations in the code examples.

## Related Issue

#1925

### CODE REVIEW

File: src/content/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Getting Started with Ethereum Development Using Alchemy
 description: "This is a beginners guide to getting started with Ethereum development using Alchemy, We’ll take you from signing up with Alchemy to making a command line request, to writing your first web3 script! No blockchain development experience necessary!"
 author: "Elan Halpern"
 tags:
-  ["Getting started", "web3.js", "Ethers.js", "Nodes", "Querying", "Alchemy"]
+  ["Getting started", "javascript", "Ethers.js", "Nodes", "Querying", "Alchemy"]
```
Comment (samajammin) [inline src/content/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md:6]: Noticing that the case of our tags are creating duplicates, e.g. `Querying` in this post vs `querying` elsewhere:
Image 2020-12-01 at 8 30 20 AM
We should establish a standard for this. Any thoughts? Title case? Sentence case?

File: src/content/developers/tutorials/using-websockets/index.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+lang: en
+sidebar: true
+tags:
+  [
+    "Alchemy",
+    "WebSockets",
+    "Querying",
+    "getting started",
+    "subscription",
+    "javascript",
```
Comment (samajammin) [inline src/content/developers/tutorials/using-websockets/index.md:14]:
```suggestion
    "JavaScript",
```

File: src/content/developers/tutorials/using-websockets/index.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+-X POST
+-H "Content-Type: application/json"
+-d '{"id": 1, "method": "eth_unsubscribe", "params": ["0x9cef478923ff08bf67fde6c64013158d"]}'
+
+
+```
+
+**Result**
+
+```
```
Comment (samajammin) [inline src/content/developers/tutorials/using-websockets/index.md:245]:
```suggestion
```json
```

**Review by samajammin — CHANGES_REQUESTED** [review]
Looks good! Let's decide on a convention for tags though - see my comment.

**Comment by ryancreatescopy** [comment]
@samajammin made tags lowercase – please review again

**Review by samajammin — APPROVED** [review]

**Review by samajammin — APPROVED** [review]

**Pull request merged**